asmcli/lib/installation_dependencies.sh

```diff
@@ -389,14 +389,17 @@ register_cluster() {
 }
 
 add_cluster_labels(){
-  local LABELS; LABELS="$(get_cluster_labels)";
-  local WANT; WANT="$(mesh_id_label)"
-  local NOTFOUND; NOTFOUND="$(find_missing_strings "${WANT}" "${LABELS}")"
-
   local PROJECT_ID; PROJECT_ID="$(context_get-option "PROJECT_ID")"
   local CLUSTER_NAME; CLUSTER_NAME="$(context_get-option "CLUSTER_NAME")"
   local CLUSTER_LOCATION; CLUSTER_LOCATION="$(context_get-option "CLUSTER_LOCATION")"
 
+  # temp workaround for attached clusters
+  if [[ -z "${CLUSTER_LOCATION}" ]]; then return 0; fi
```

Comment:
Is it guaranteed to be unset only for attached clusters? Regular Anthos clusters don't seem have the issue despite failing to get labels and having CLUSTER_LOCATION unset, is there some interplay with `register_cluster`? In logs I have it appears like it returns on line 405.

Reply:
I'm 90% sure yes, and I'm 100% sure that attached clusters don't support the features that cluster labels enable so this shouldn't affect anything.